sdk/ml/azure-ai-ml/tests/dsl/e2etests/test_automl_dsl_pipeline.py

```diff
@@ -59,9 +63,10 @@ def train_automl_classification_in_pipeline(
         )
 
         pipeline_job: PipelineJob = train_automl_classification_in_pipeline(class_train, class_valid)
+        pipeline_job.settings.default_compute = "cpu-cluster"
```

Comment:
may add a variable for default_compute

Reply:
added for `cpu-cluster` and `gpu-cluster`